src/org-pending/store/actions/org-pending.actions.spec.ts

```diff
@@ -1,6 +1,20 @@
 import * as fromPendingOrganisation from './org-pending.actions';
 import { PendingOrganisation } from 'src/org-pending/models/pending-organisation';
 
+const payload: PendingOrganisation[] = [
+  {
+    name: 'Glen Byrne',
```

Comment:
What do we feel about a fixtures folder for all the json data inline with the tests? Data can be removed and used as an import to make tests cleaner and a central place for them allows them to be reused?

Reply:
All these mocks are now under `src/org-pending/mock/pending-organisation.mock.ts`